src/main/groovy/de/dkfz/roddy/execution/jobs/GenericJobInfo.groovy

```diff
@@ -29,37 +56,26 @@ class GenericJobInfo {
     BufferUnit memoryBufferUnit
     String queue
     String otherSettings
-    String user;
-    String subHost; //submission host
-    String exHosts; // execution hosts
-    String runTime; //Time in seconds that the job has been in the run state
-    String subTimeGMT;
-    String startTimeGMT;
-    String endTimeGMT;
+    JobState jobState
     String numProcessors;
-    String cpuTime; //Cumulative total CPU time in seconds of all processes in a job
     String userTime; //user time used
     String systemTime; //system time used
     String runLimit;
     String pendReason;
-    String priority;
-    String userGroup;
-    String resReq; // resource requirements
     String execHome;
     String execUserName;
     String pidStr;
     String pgidStr;
     String nthreads; //Number of currently active threads of a job
     String swap; //Total virtual maxMemory (swap) usage of all processes in a job
-    String exitStatus; // UNIX exit status of the job
+    String exitCode; // UNIX exit status of the job
     String jobGroup;
     String description;
     String execCwd; //Executed current working directory
     String askedHostsStr;
     String cwd; //Current working directory
     String projectName;
-    String outfile;
-    String infile;
+
     String timeUserSuspState; //Suspended by its owner or the LSF administrator after being dispatched
     String timePendState; //Waiting in a queue for scheduling and dispatch
     String timePendSuspState; // Suspended by its owner or the LSF administrator while in PEND state
```

Comment:
These should be type Duration, you could also rename the variables (time… -> duration…) to distinguish them from time points